src/Engines/IChannel.cs

```diff
@@ -75,5 +77,11 @@ public IUpdatableDisplay DisplayUpdatable(object displayable)
             this.Display(displayable);
             return new UpdatableDisplayFallback(this);
         }
+
+        /// <summary>
+        ///     Sends an iopub message to the client associated with this channel.
+        /// </summary>
+        /// <param name="message">The message to send. Cannot be null.</param>
+        void SendIoPubMessage(Message message);
```

Comment:
Since this is a breaking change as written (though in practice engines should rely on the BaseEngine implementation of IChannel), should we mitigate that by having a default implementation that throws `NotImplementedException`?

Reply:
Sure, that makes sense. As you point out, probably doesn't matter too much, but seems like a good practice.